docs/conf.py

```diff
@@ -31,6 +31,7 @@
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
+    'sphinx.ext.doctest',
```

Comment:
Are we also going to have `doctest` scan these things via the usual test runner? Using this will only work when making the documentation but ideally we'd want to make sure the docs are correct before we get to that stage.